joblibspark/backend.py

```diff
@@ -137,11 +145,31 @@ def _get_pool(self):
             self._pool = ThreadPool(self._n_jobs)
         return self._pool
 
+    def start_call(self):
+        self._is_running = True
+        if self._ipython is not None:
+            def on_post_run_cell(result):
+                if result.error_in_exec is not None:
+                    self._cancel_all_jobs()
+
+            self._on_post_run_cell = on_post_run_cell
+            self._ipython.events.register("post_run_cell", on_post_run_cell)
+
+    def stop_call(self):
+        if self._ipython is not None:
+            self._ipython.events.unregister("post_run_cell", self._on_post_run_cell)
+
     def apply_async(self, func, callback=None):
         # Note the `func` args is a batch here. (BatchedCalls type)
         # See joblib.parallel.Parallel._dispatch
 
+        ipython_command_canceled = False
```

Comment:
How does `ipython_command_canceled` become `True`?

Reply:
fixed. ipython_command_canceled is not used